fdbserver/workloads/Mako.actor.cpp

```diff
@@ -1,35 +1,34 @@
 #include "fdbclient/NativeAPI.actor.h"
 #include "fdbserver/TesterInterface.actor.h"
-#include "workloads.actor.h"
+#include "fdbserver/workloads/workloads.actor.h"
 #include "fdbserver/workloads/BulkSetup.actor.h"
 #include "fdbclient/ReadYourWrites.h"
 #include "flow/actorcompiler.h"
 
 
 enum {OP_GETREADVERSION, OP_GET, OP_GETRANGE, OP_SGET, OP_SGETRANGE, OP_UPDATE, OP_INSERT, OP_INSERTRANGE, OP_CLEAR, OP_SETCLEAR, OP_CLEARRANGE, OP_SETCLEARRANGE, OP_COMMIT, MAX_OP};
 enum {OP_COUNT, OP_RANGE};
-static constexpr int MAXKEYVALUESIZE = 1000;
-static constexpr int KEYPREFIXLEN = 4;
-static constexpr int GETRANGELIMIT = 10000;
-static const char* opNames[MAX_OP] = {"GRV", "GET", "GETRANGE", "SGET", "SGETRANGE", "UPDATE", "INSERT", "INSERTRANGE", "CLEAR", "SETCLEAR", "CLEARRANGE", "SETCLEARRANGE", "COMMIT"};
-static const std::string KEYPREFIX = "mako";
+constexpr int MAXKEYVALUESIZE = 1000;
+constexpr int RANGELIMIT = 10000;
 struct MakoWorkload : KVWorkload {
 	uint64_t rowCount, seqNumLen, sampleSize, actorCountPerClient;
 	double testDuration, loadTime, warmingDelay, maxInsertRate, transactionsPerSecond, allowedLatency, periodicLoggingInterval;
 	bool enableLogging, commitGet, populateData, runBenchmark, preserveData;
 	PerfIntCounter xacts, retries, conflicts, commits, totalOps;
-	vector<PerfIntCounter> opCounters;
-	vector<uint64_t> insertionCountsToMeasure;
-	vector<std::pair<uint64_t, double>> ratesAtKeyCounts;
+	std::vector<PerfIntCounter> opCounters;
+	std::vector<uint64_t> insertionCountsToMeasure;
+	std::vector<std::pair<uint64_t, double>> ratesAtKeyCounts;
 	std::string valueString, operationsSpec;
-
 	//store operations to execute
 	int operations[MAX_OP][2];
-
 	// used for periodically tracing
-	vector<PerfMetric> periodicMetrics;
-	Standalone<VectorRef<ContinuousSample<double>>> opLatencies;
+	std::vector<PerfMetric> periodicMetrics;
+	// store latency of each operation with sampling
+	std::vector<ContinuousSample<double>> opLatencies;
 
+	static inline const std::string KEYPREFIX = "mako";
+	static inline const int KEYPREFIXLEN = KEYPREFIX.size();
+	static inline const std::array<std::string, MAX_OP> opNames = {"GRV", "GET", "GETRANGE", "SGET", "SGETRANGE", "UPDATE", "INSERT", "INSERTRANGE", "CLEAR", "SETCLEAR", "CLEARRANGE", "SETCLEARRANGE", "COMMIT"};
```

Comment:
I honestly don't think I was aware that you could declare a variable as inline.  I'd recommend removing it, as it'd be permitting ODR violations, which ... shouldn't be an issue because of `static`, but I'd prefer to not have to think about it.

Reply:
I am sorry I misunderstood your meaning. I have removed the `static` and thus these will not be in the binary.